### PR TITLE
Bugfix/nw23001440/check bif

### PR DESCRIPTION
**Pull request by cosentino-smeup**
## Description
In this branch we have implemented the BIF %CHECK

Note: 
the fourth parameter has not been included:
{ : *NATURAL | *STDCHARSIZE}

Related to:
T04_A15

## Checklist:
- [X] There are tests regarding this feature
- [X] The code follows the Kotlin conventions (run `./gradlew ktlintCheck`)
- [X] The code passes all tests (run `./gradlew check`)
- [ ] There is a specific documentation in the `docs` directory

### CODE REVIEW

File: rpgJavaInterpreter-core/src/test/resources/smeup/T04_A15.rpgle
Comment (lanarimarco) [inline rpgJavaInterpreter-core/src/test/resources/smeup/T04_A15.rpgle:1]: Why not a single program per step?

**Review by lanarimarco — CHANGES_REQUESTED** [review]
When you implement a new bif (and in general it is always a good practice) you should add at least a `BIFCHECK.rpgle` test program in `rpgJavaInterpreter-core/src/test/resources` path and the related function  `InterpreterTest.executeBIFCHECK`.
The `Txx_Ayy_Pzz` tests often contain a lot of noise and it is not always clear what they  are testing.

File: rpgJavaInterpreter-core/src/test/resources/smeup/T04_A15.rpgle
Comment (cosentino-smeup) [reply rpgJavaInterpreter-core/src/test/resources/smeup/T04_A15.rpgle:1]: added BIFCHECK

Comment (cosentino-smeup) [reply rpgJavaInterpreter-core/src/test/resources/smeup/T04_A15.rpgle:1]: split T04_15 into 3 tests

**Review by lanarimarco — APPROVED** [review]
Ok

**Pull request merged**